source/compute/portal/ondemand/matlab.rst

```diff
@@ -0,0 +1,63 @@
+.. _ood_matlab:
+
+MATLAB
+------
+
+Who can use MATLAB
+~~~~~~~~~~~~~~~~~~
+
+.. tab-set::
+
+   .. tab-item:: KU Leuven/UHasselt
+
+      Only vsc3* users (affiliated with KU Leuven) who are members of the
+      ``lli_matlab`` group have rights to use the MATLAB module (hence the
+      MATLAB app). If you are not already member of the group, contact the
+      :ref:`KU Leuven supprt team<user support VSC>` for an invitation, or
+      :ref:`request joining this group<join groups>` via your VSC account page.
+
+   .. tab-item:: VUB
+
+      Only vsc1* users (affiliated with VUB) who are members of the
+      ``brusselall`` group have rights to use the MATLAB app.
+
+Before you connect
```

Comment:
This issue with the .MathWorks directory specifically happens for Matlab 2024a (before Update 6), and is caused by a separate installation of Mathworks Service Host for each of the nodes Matlab is launched on by the user. In newer versions the size is more limited as these installations (should) only use a single MathWorks Service Host installation. We are currently also implementing a fix for a central installation of this software. Feel free to contact me to explain the fix. 

This also means that you can put this part under a VUB-specific tab.